yt_dlp/extractor/ard.py

```diff
@@ -601,3 +602,186 @@ def fetch_page(page_num):
         return self.playlist_result(
             OnDemandPagedList(fetch_page, self._PAGE_SIZE), full_id, display_id=display_id,
             title=page_data.get('title'), description=page_data.get('synopsis'))
+
+
+class ARDAudiothekBaseIE(InfoExtractor):
+    _GRAPHQL_ENDPOINT = 'https://api.ardaudiothek.de/graphql'
+
+    def _graphql_query(self, urn, query):
+        return self._download_json(
+            self._GRAPHQL_ENDPOINT,
+            urn,
+            data=json.dumps({'query': query,
+                             'variables': {'id': urn},
+                             }).encode(),
+            headers={
+                'Content-Type': 'application/json',
+            },
+        )['data']
```

Comment:
I would inline the URL since it's only used once
```suggestion
    def _graphql_query(self, urn, query):
        return self._download_json(
            'https://api.ardaudiothek.de/graphql', urn,
            data=json.dumps({
                'query': query,
                'variables': {'id': urn},
            }).encode(), headers={
                'Content-Type': 'application/json',
            })['data']

```